zerver/openapi/zulip.yaml

```diff
@@ -6729,16 +6742,30 @@ paths:
           content:
             application/json:
               schema:
-                allOf:
-                  - $ref: "#/components/schemas/CodedError"
-                  - example:
-                      {
-                        "result": "error",
-                        "msg": "Invalid message(s)",
-                        "code": "BAD_REQUEST",
-                      }
-                    description: |
-                      An example JSON error response for when the emoji code is invalid:
+                oneOf:
```

Comment:
Hmm, should this `oneOf` block still have the a generic `BAD_REQUEST` variant? I guess at present we don't document the totally generic options, like "missing authentication headers" or even the fairly generic ones, like "Invalid message(s)", just the interesting cases for this specific endpoint, so probably not super relevant for this PR.

(I guess I'm not sure how we're handling this in general in the API docs).

Reply:
Yeah, there was even an error that this change cleaned up here in the `/api/remove-reaction` error documentation in that it had the error message as "Invalid message(s)" and not "Invalid emoji code".

In general, I think an #api documentation discussion for the error documentation is something it would be good for me to move up on the list.